src/includes/configuration/auto-session-tracking/android.mdx

```diff
@@ -1,3 +1,26 @@
+By default, the session is terminated once the application is in the background for more than 30 seconds. You can change the time out with the option named `sessionTrackingIntervalMillis`. It takes the amount in milliseconds. For example, to configure it to be 60 seconds:
+
+```xml {filename:AndroidManifest.xml}
+<application>
+  <meta-data android:name="io.sentry.session-tracking.timeout-interval-millis" android:value="10000" />
+</application>
+```
+
+Or, if you are manually instrumenting Sentry:
+
+```java
+import io.sentry.android.core.SentryAndroid;
+
+SentryAndroid.init(
+  options -> {
+    options.setSessionTrackingIntervalMillis(10000);
+  });
```

Comment:
the text says:

> For example, to configure it to be 60 seconds

but the code snippet os 10000 = 10s 

either fix the code snippets or change the text

Reply:
Fixed